src/components/Calendar.tsx

```diff
@@ -36,14 +43,14 @@ const MAX_DOTS_PER_DAY = 5;
 
 setupLocales();
 
-const getMarkedDates = (data?: any[]) => {
+const getMarkedDates = (data?: any[], dotCount?: number, selectedDay: string) => {
   const markedDates: CalendarProps['markedDates'] = {};
 
   data?.forEach((item) => {
     if (
       !!item.listDate &&
       (!markedDates?.[item?.listDate]?.dots ||
-        markedDates?.[item?.listDate]?.dots?.length < MAX_DOTS_PER_DAY)
+        markedDates?.[item?.listDate]?.dots?.length < (dotCount || MAX_DOTS_PER_DAY))
```

Comment:
please use a default in the method params instead of the || here

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/6f91b33f23931efd529ad20b36df665c327042ca